packages/origin-backend/src/pods/device/device.service.ts

```diff
@@ -185,6 +185,24 @@ export class DeviceService {
         return devices;
     }
 
+    async getMyDevice(organizationId: number, options: FindOneOptions<Device> = {}) {
```

Comment:
You don't need to create a new duplicate method for this, you can just use `getAll()`

```getAll({ where: { organization: { id: organizationId } } });```